### ISSUE TITLE
Disable lock when modifier key is pressed?

### ISSUE THREAD

**Issue by markchalloner** [shtirlic/yubikeylockd#7]
Sometimes I need to leave my laptop unlocked when removing the Yubikey (e.g. when transferring it a short distance and unplugging the USB hub). Would it be possible to add a check for a modifier key (e.g. CTRL) is not pressed before locking?

Thanks

**Comment by shtirlic** [comment]
Yes, sometimes I need this too, but from my point of view this method is destroying security via "keylogging" modifier key.
